src/graphql/internal.rs

```diff
@@ -67,3 +91,97 @@ struct InternalGraphQLResponse<T> {
     data: Option<T>,
     errors: Option<serde_json::Value>,
 }
+
+#[derive(Deserialize, Serialize, Debug, Clone)]
```

Comment:
```suggestion
#[derive(Default, Deserialize, Serialize, Debug, Clone)]
```

Reply:
I intentionally did not derive Default, as I wanted the user to always put in a user pool in the `new` function. What are your thoughts on that?

Reply:
Yeah, i kinda figured that much which is also why the suggestion was part of "approved" :p It makes sense as long as it is reasoned.